ansible/test.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Playbook for testing individual and sets of roles.
+#
+# Host names in this file should match the name of the Vagrant configuration
+# in Vagrantfile. Tests can be run either by using `vagrant up $NAME` when a
+# box does not yet exist, or `vagrant provision $NAME` for one that exists &
+# is running.
+
+- hosts: docker
+  sudo: True
```

Comment:
Will do. I tried for a larger docker configuration which *did* require `sudo: True` so likely we're missing some `become: yes` lines in one of: docker-storage, docker-advanced, docker-dns-*